src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -80,6 +81,14 @@ export class CreateClusterComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
+    this.route.queryParams.subscribe(params => {
+      // reading 'welcome' value true/false to toggle expand-cluster wizand view and welcome view
+      const showWelcomeScreen = params['welcome'];
+      if (!_.isEmpty(showWelcomeScreen)) {
```

Comment:
just checking for a simple string empty check do we need to do it using the `isEmpty` or can we just rely on the below
```suggestion
      if (showWelcomeScreen) {
```

Reply:
yeah, we can omit that